src/components/Checkbox/__tests__/checkbox-tests.jest.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { fireEvent, render, cleanup, screen } from "@testing-library/react";
+import { expect as sinonExpect } from "../../../test/test-helpers";
+import Checkbox from "../Checkbox";
+
+describe("Checkbox tests", () => {
```

Comment:
please add a test that validates the onChange expected behavior(https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) specifically the `event.target.value`

Reply:
unfortunately, it can be possible only for React 17 (https://reactjs.org/blog/2020/08/10/react-v17-rc.html#no-event-pooling)

Reply:
@orrgottlieb can I mark this as resolved?